source/content/composer.md

```diff
@@ -54,9 +54,11 @@ To maintain a Composer-based workflow with one-click updates through the Dashboa
 
 Use [Integrated Composer](/integrated-composer) to preserve the functionality of Pantheon's one-click updates from the Site Dashboard for Composer-managed sites created from a [Custom Upstream](/custom-upstream).
 
-A Custom Upstream based off Pantheon's example repositories would need to commit all dependencies. Updates via Composer would only happen at the Custom Upstream repository level by a single repository maintainer. Those updates would then trickle down to sites created from the Custom Upstream as one-click updates in the Pantheon Site Dashboard.
+Updates made using Integrated Composer are only committed at the Custom Upstream level by a single repository maintainer. Updates then trickle down to sites created from the Custom Upstream as one-click updates in the Pantheon Site Dashboard. Custom Upstreams using Integrated Composer with dependencies in `upstream-configuration/composer.json` can have their own site-specific dependencies added in the top-level `composer.json` file. Custom Upstreams using Integrated Composer should avoid changing the top-level `composer.json` file after sites have been created from the Custom Upstream to avoid merge conflicts.
```

Comment:
@whitneymeredith I wanted to make sure this bit is correct: `Custom Upstreams using Integrated Composer should avoid changing the top-level `composer.json` file after sites have been created from the Custom Upstream to avoid merge conflicts.` 

Because we are differentiating, I wondered if this should read: `Custom Upstreams **that don't use** Integrated Composer should avoid changing the top-level `composer.json` file after sites have been created from the Custom Upstream to avoid merge conflicts.`

Let me know your thoughts!

Reply:
@michellecolon-pantheon  I believe the original edit is correct based on the suggested solution: "An upstream based on Integrated Composer should avoid ever changing the top-level composer.json file after release (after sites have been created from the upstream) to avoid causing merge conflicts." Unless I'm reading this wrong??

Reply:
@whitneymeredith Ah, no, you are reading that correctly. I was asking because it looks like we're trying to let folks know what needs to be done when using Integrated Composer and what needs to be done when not using Integrated Composer, but I only see Integrated Composer-specific directions. Unless I missed something in there?

Reply:
@michellecolon-pantheon Line 59 addresses if you're not using Integrated Composer.

Reply:
@whitneymeredith Ah, okay. For whatever reason, I thought there would be something about it in this line 🙃  Thanks!